protobuf/echo/Echo.hpp

```diff
@@ -207,7 +208,6 @@ namespace services
 
     private:
         Echo& echo;
-        uint32_t serviceId;
```

Comment:
if serviceId is not necessary, then the parameter to the constructor should be removed completely

Reply:
I have removed it completely. Including from the code generator.